CONTRIBUTING.md

```diff
@@ -323,9 +325,13 @@ For all OS types perform the following:
 #### RHEL 7/8
 
 1. Run `pyinstaller -F ./pkg_bin/conjur`. Once this is run, a `dist` folder will be created with the executable in it.
-1. Once an executable has been created, run `zip conjur-cli-rhel-7 conjur` (`zip conjur-cli-rhel-8 conjur` for RHEL 8 
-   machines) to zip the file.
-1. Sign the `zip` and add it as an asset in the release page.
+1. Once an executable has been created, archive the file for RHEL 7 and RHEL 8 platforms, using the following commands:
+  1. `tar cvf conjur-cli-rhel-7.tar.gz conjur`
```

Comment:
why do we need `c` instead of `x`? We don't want to create a new folder, instead we just want to extract the contents of the tar to the directory where that command was run

Reply:
on my RHEL 7 machine, I see the following error
```
$ tar cvf conjur-cli-rhel-7.tar.gz conjur
tar: conjur: Cannot stat: No such file or directory
tar: Exiting with failure status due to previous errors
```

Reply:
i don't follow. you have the tar and you want to extract it? if so - you need to run `tar xvf` and not `tar xvf`

Reply:
```
tar xvf conjur-cli-rhel-7.tar.gz

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

Reply:
seems the exec was corrupted. fixed with @orenbm